devops/github_issue_alert/program.py

```diff
@@ -0,0 +1,38 @@
+"""Monitors GitHub issue creation and comment activity.
+
+Sending the notifications to Slack to keep teams updated.
+"""
+
+import os
+
+from autokitteh.slack import slack_client
+
+
+SLACK_CHANNEL = os.getenv("SLACK_CHANNEL", "")
+
+slack = slack_client("slack_conn")
+
+
+def on_issue_comment(event):
```

Comment:
Document these functions.

Also be specific about which `action` values you're supporting ("all of them" is great, but only if you actually tested all of them).

Also, even though it's not a general convention we have, it would be helpful in this case for users who read this code to see a link to GitHub's documentation of these events:

https://docs.github.com/en/webhooks/webhook-events-and-payloads#issue_comment
https://docs.github.com/en/webhooks/webhook-events-and-payloads#issues